src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -68,6 +71,15 @@
 @ImplementsService(name = ClientDevicesAuthService.CLIENT_DEVICES_AUTH_SERVICE_NAME)
 public class ClientDevicesAuthService extends PluginService {
     public static final String CLIENT_DEVICES_AUTH_SERVICE_NAME = "aws.greengrass.clientdevices.Auth";
+    private static final ObjectMapper MAPPER = JsonMapper.builder()
```

Comment:
use the mappers we already have in the utils. We have one already for recipes which does case insensitive stuff